met-api/src/met_api/services/survey_service.py

```diff
@@ -35,21 +36,21 @@ def get_open(cls, survey_id):
         return survey
 
     @staticmethod
-    def get_surveys_paginated(pagination_options: PaginationOptions, search_text='', unlinked=False,
-                              exclude_hidden=False):
+    def get_surveys_paginated(pagination_options: PaginationOptions, survey_exclusion_options: SurveyExclusionOptions,
```

Comment:
we might want to move the search text and unlinked filters into the "searchObject". its just an idea but no need to implement